examples/user_interfaces/toolmanager.py

```diff
@@ -83,10 +114,21 @@ def set_lines_visibility(self, state):
 fig.canvas.manager.toolbar.add_tool('zoom', 'foo')
 
 # Remove the forward button
-fig.canvas.manager.toolmanager.remove_tool('forward')
+tool_mgr.remove_tool('forward')
 
 # To add a custom tool to the toolbar at specific location inside
 # the navigation group
 fig.canvas.manager.toolbar.add_tool('Hide', 'navigation', 1)
 
+for i, c in enumerate(['yellowgreen', 'forestgreen']):
+    sidebar = fig.canvas.manager._get_toolbar()
```

Comment:
Also as this example demonstrates the new toolmanager class, it has ``rcParams['toolbar'] = toolmanager`` at the beginning :wink:.

Speaking generally, only the adding of tools and setting the flow aka orientation of the toolbar will fail, everything else will just work (i.e. the creation of the new toolbar and adding it to the western edge of the window).